.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,21 @@
+name: DockerBuild
+
+#on: push
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
These needed to be added in the `Secrets` sections of the repository settings. I currently do not have permission to add them myself. You'll need to go to the [Docker Hub Security Settings](https://hub.docker.com/settings/security) and create a new access token to use as `DOCKER_PASSWORD`.

Reply:
You should be able to do this now.

Reply:
They have been added.